### ISSUE TITLE
Serverless_negs module: Quic - No option to disable HTTP/3 advertising

### ISSUE THREAD

**Issue by sakthivel-tw** [terraform-google-modules/terraform-google-lb-http#238]
### TL;DR

With the recent communication from GCP, `quicOverride=NONE` mode will advertise support for HTTP/3 to our clients. Providing Terraform config `quic = false` is updated as `quicOverride=NONE`. Deadline for the rollout is March 21, 2022.

A new option is required to disable HTTP/3 advertisement or by default `quic = false` should disable the HTTP/3 mode. Possibly change the data type from boolean to string to support different options.

https://cloud.google.com/load-balancing/docs/https#configuring-http3

<img width="851" alt="Screenshot 2022-03-14 at 7 17 42 PM" src="https://user-images.githubusercontent.com/44604719/158187756-f0f9f8a7-8c8d-4da3-b4df-1dce293ea99e.png">

### Terraform Configuration

```hcl
quic = false
```

Doesn't change it to `quicOverride=DISABLED`.
```


### Terraform Version

Terraform v0.14.3
+ provider registry.terraform.io/hashicorp/google v3.90.1
+ provider registry.terraform.io/hashicorp/google-beta v3.90.1

Your version of Terraform is out of date! The latest version
is 1.1.7. You can update by downloading from https://www.terraform.io/downloads.html


### Additional information

_No response_

**Comment by imrannayer** [comment]
@sakthivel-tw Setting quic = false should send null value. Provider default value is `None` which is equivalent of `DISABLE`. Can you plz confirm if this is working fine?

**Comment by yu-ichiro** [comment]
@imrannayer The Provider default value `NONE` is acting as the same as `ENABLE`. (I can confirm on my project HTTP/3 is advertized when set to `NONE`)
<img width="481" alt="image" src="https://user-images.githubusercontent.com/6957492/231441261-10288470-24fe-4b07-82e4-7b1309f72374.png">

I think making `false` map to `DISABLE` will retain the original behaviour prior to March 21, 2022
